clientconn.go

```diff
@@ -1166,7 +1158,8 @@ func (ac *addrConn) adjustParams(r transport.GoAwayReason) {
 
 func (ac *addrConn) resetTransport() {
 	ac.mu.Lock()
-	if ac.state == connectivity.Shutdown {
+	acCtx := ac.ctx
```

Comment:
Since `ac.ctx` is now re-assigned in `updateAddresses`, all accesses need to be guarded, right?

I see one here that is probably not guarded: https://github.com/grpc/grpc-go/blob/098b2d00c5bcbc5c696dcab9a2a7f7b442f8142e/stream.go#L1278

Reply:
Good catch, thanks!